shared/src/types/address.rs

```diff
@@ -531,6 +532,16 @@ pub fn kartoffel() -> Address {
     Address::decode("atest1v4ehgw36gep5ysecxq6nyv3jg3zygv3e89qn2vp48pryxsf4xpznvve5gvmy23fs89pryvf5a6ht90").expect("The token address decoding shouldn't fail")
 }
 
+/// Temporary helper for testing
+pub const fn wnam() -> EthAddress {
+    // TODO: Replace this with the real wNam ERC20 address once it exists
+    // "DEADBEEF DEADBEEF DEADBEEF DEADBEEF DEADBEEF"
+    EthAddress([
+        222, 173, 190, 239, 222, 173, 190, 239, 222, 173, 190, 239, 222, 173,
+        190, 239, 222, 173, 190, 239,
+    ])
+}
```

Comment:
We can remove the TODO and could move this helper fn to `test_bridge_pool_vp` since it shouldn't be used in production now.